gpustack/policies/candidate_selectors/sglang_resource_fit_selector.py

```diff
@@ -689,6 +784,201 @@ async def auto_select_multi_worker_multi_gpu_candidates(
 
         return []
 
+    async def _cal_mem_fraction_static(self, workers: List[Worker]):  # noqa: C901
+        """
+        Adapted from sglang's server_args memory fraction logic.
```

Comment:
Place the reference code link here and list the changes we have made compared to the original version.